packages/kbn-management/cards_navigation/src/consts.tsx

```diff
@@ -97,6 +97,14 @@ export const appDefinitions: Record<AppId, AppDefinition> = {
     icon: 'wrench',
   },
 
+  [AppIds.SPACES]: {
+    category: appCategories.CONTENT,
+    description: i18n.translate('management.landing.withCardNavigation.spacesDescription', {
+      defaultMessage:
+        'Organize your dashboards and other saved objects into meaningful categories.',
```

Comment:
```suggestion
        'Organize saved objects and set visible features for different groups of users.',
```
Trying to stay concise, but would that be a little closer to the way spaces currently work? I think the current definition hasn't changed since the feature was introduced in v6.something

Reply:
I just discussed with my team. We actually hide the feature visibility toggles for serverless spaces. Administrators are expected to configure privileges instead via custom roles. And we want to be careful not to imply that spaces provides a way to group users. So I will just shorten the original statement to 'Organize your saved objects into meaningful categories.'